### PR TITLE
Updated dark theme styling

### PR DESCRIPTION
**Pull request by pe4cey**
Includes: 

- `:play` frames
- `:sysinfo`
- graph results frame
- fullscreen frames

### CODE REVIEW

File: app/styles/themes/dark.styl [hunk 1]
```diff
@@ -21,30 +21,47 @@ tertiary-background-hover = #292e3a
...
 color-link-hover = #ffffff
-
-
 body.theme-dark
   font: 13px 'Helvetica Neue', Helvetica, Arial, sans-serif
   background: primary-background-color
   color: primary-text
-
+  .outline
+    stroke: primary-background-color
```
Comment (sendilkumarn) [inline app/styles/themes/dark.styl]: this should be `primary-text-color` right?

Comment (pe4cey) [inline app/styles/themes/dark.styl]: For the `dark` theme it make more sense to go with a darker outline.

**Comment by pe4cey** [comment]
I've just pushed a commit (after looking at @sendilkumarn's comments) to keep the stroke colour the same for both nodes and relationships

**Comment by sendilkumarn** [comment]
👍  I was confused by the background and the below one [ which you changed just now ] 

Table transparent, is not quite good in dark mode. Can we have something slightly darker there, if possible?

**Pull request merged**